1/assets/1.2-finished-code.rs

```diff
@@ -3,28 +3,30 @@
 
 use ink_lang as ink;
 
-#[ink::contract(version = "0.1.0")]
+#[ink::contract()]
```

Comment:
```suggestion
#[ink::contract]
```

Reply:
Please remove the empty parentheses `()` for all code examples. Only use parentheses when they actually enclose parameters.